cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -18,31 +18,29 @@
 
 {% macro _secondary_navigation(items, active_item_id) %}
 {% from 'molecules/nav-link.html' import nav_link as nav_link %}
-<ul class="o-secondary-navigation_list
-           o-secondary-navigation_list__parents">
-    {%- for item in items %}
-        {%- set href, id, caption, children = item[0], item[1], item[2], item[3:] %}
-
-    <li>
-        {{ nav_link(caption, href, true, id == active_item_id) }}
-    {%- if children -%}
-        {% set children = children[0] %}
-        <ul class="o-secondary-navigation_list
-                   o-secondary-navigation_list__children">
-        {%- for href, id, caption in children %}
+    <ul class="o-secondary-navigation_list
+               o-secondary-navigation_list__parents">
+        {%- for item in items %}
             <li>
-                {{ nav_link(caption, href, false, id == active_item_id) }}
+                {{ nav_link(item.title, item.url, true, item.url == request.url) }}
+                {%- for child in item.children -%}
+                    <ul class="o-secondary-navigation_list
+                               o-secondary-navigation_list__children">
+                        <li>
+                            {{ nav_link(child.title, child.url, false, child.url == request.url) }}
+                        </li>
+                    </ul>
+                {%- endfor %}
             </li>
-        {%- endfor %}
-        </ul>
-    {%- endif %}
-    </li>
 
-    {%- endfor %}
-</ul>
+        {%- endfor %}
+    </ul>
 {% endmacro %}
 
-<nav class="o-secondary-navigation"
+{% set nav_items, has_children = get_secondary_nav_items(page) %}
+
+<nav class="o-secondary-navigation
+            {{ 'u-hide-on-mobile' if not has_children else '' }}"
```

Comment:
I've realized we need to hide this on tablets as well. If you add a TODO here, I'll pick this up after it's merged, just to keep it on scope.